internal/logger/target/http/http.go

```diff
@@ -361,9 +356,8 @@ func (h *Target) startQueueProcessor(ctx context.Context, mainWorker bool) {
 			return
 		}
 
-		if !isTick {
+		if count < h.batchSize {
```

Comment:
I think we can miss some events here.. we are receiving an entry from h.logCh and globalBuffer then we have this condition that will ignore adding this entry to the buffer or entries array